### PR TITLE
Changes in plugin settings

### PR DESCRIPTION
**Pull request by KELiON**
* Added "settings" button to plugin preview
* Unify usage of form component in plugin settings and in global settings
* Simplify form components API
* Fix tests/linters

### CODE REVIEW

File: app/lib/plugins/settings/index.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import get from './get'
 import validate from './validate'
-import add from './add'
 
-export default { get, validate, add }
```
Comment (BrainMaestro) [inline app/lib/plugins/settings/index.js:5]: We don't need this?

Comment (KELiON) [reply app/lib/plugins/settings/index.js:5]: @BrainMaestro  There are get and validate, I don't see meaning of `add` method

Comment (BrainMaestro) [reply app/lib/plugins/settings/index.js:5]: But neither of those actually add the settings to the config. Does this work for you while testing with a plugin that defines some settings? Maybe I'm missing something.

Comment (KELiON) [reply app/lib/plugins/settings/index.js:5]: @BrainMaestro but what do you mean by add to config? We save changes values to global config [here](https://github.com/KELiON/cerebro/blob/7d8b75702e79668b719afef1aa46f295a6d4c13e/app/main/plugins/core/cerebro/plugins/Preview/Settings.js#L39)

Comment (BrainMaestro) [reply app/lib/plugins/settings/index.js:5]: Yeah my bad

**Pull request merged**